Tests/Functional/App/config/exception/config.yml

```diff
@@ -7,7 +7,7 @@ overblog_graphql:
         class_namespace: "Overblog\\GraphQLBundle\\Exception\\__DEFINITIONS__"
         exceptions:
             errors:
-              - "InvalidArgumentException"
+              - 'InvalidArgumentException'
```

Comment:
Is this necessary? Feels like we are now mixing single and double quotes, maybe we can choose one

Reply:
nope no necessary, forget to rollback this